scripts/move.sh

```diff
@@ -4,7 +4,7 @@ pushd $(dirname "$0")/..
 
 export RPC_URL="http://localhost:5050";
 
-export WORLD_ADDRESS=$(cat ./manifests/dev/manifest.json | jq -r '.world.address')
+export WORLD_ADDRESS=$(cat ./manifests/dev/deployment/manifest.json | jq -r '.world.address')
 
 # sozo execute --world <WORLD_ADDRESS> <CONTRACT> <ENTRYPOINT>
-sozo execute --world $WORLD_ADDRESS dojo_starter::systems::actions::actions move -c 1 --wait
+sozo execute --world $WORLD_ADDRESS dojo_starter::systems::actions::dojo_starter-actions move -c 1 --wait
```

Comment:
We should actually remove those scripts and let's just have some README with the command which is now simpler:
```
sozo execute actions move -c 1 --wait
```
The world is not expected to be passed as taken from the dojo env. 👍